### DISCUSSION TITLE
Aaeon Upboard support

### DISCUSSION THREAD

**Discussion by FPSychotic** [adafruit/Adafruit_Blinka#1056] [New Board Requests]
Hi, Could you confirm if Blinka is supported by Upboard, the are compatible with many RPi HATs as it share the same gpio interface. To be compatible, I think that internally they use the same approach than Blinka, a software layer to correspond different physical layers in the CPU and Chipsets.
If it is not compatible could you add support for this board, and in consequence probably to the incoming Banana PiX  as they share the same hardware

**Comment by makermelissa** [comment]
@FPSychotic, I'm almost certain that it is not supported. Blinka uses PlatformDetect to figure out the processor that it is running on and the Aaeon Upboard uses an Intel Atom x5-z8350 processor and that doesn't appear to be one we have inside of https://github.com/adafruit/Adafruit_Blinka/tree/master/src/adafruit_blinka/microcontroller. I'll leave this issue open for now as a request to support that board, even though I don't have one.

**Comment by astumpf** [comment]
Still not supported? I'm wondering as the Aaeon boards are widely used for robotics application.

**Comment by ladyada** [comment]
we're happy to take a PR from someone who owns one - we do not have any plans to do it ourselves.
